### DISCUSSION TITLE
support for google forms and meet

### DISCUSSION THREAD

**Discussion by still-learner** [SafeExamBrowser/seb-win-refactoring#122] [Ideas]
support for google forms and meet so that it becomes easier for everybody since many are not using moodle or ilias.they use google classroom

**Comment by pazao** [comment]
I am using google classroom. I found out Seb through my use of Bookwidgets, but it's not free.

**Comment by dbuechel** [comment]
Thanks for your input. Could you elaborate a bit on your exact use case, i.e. what functionality you would like to see provided by SEB? And could you provide more details about the current incompatibilities, i.e. what exactly isn't working at the moment when trying to use Google Forms or Google Meet within SEB?

**Reply by still-learner** [reply]
thanks for reply , later i was able to use google forms on seb. 
on setting up for server for jitsi meet ,i am stopped at point where i have to create localhost key and certificate, should i use open ssl or somthing else and can you mention that command that will help me to open interactive window for server setup 
where should i put jitsi meet link
is there any video explaining server setup on pc
and one more thing why is this browser not available for android as many people use it and how to use seb website whose code is mentioned in repo
thanks again

**Reply by danschlet** [reply]
The Jitsi Meet proctoring features are still experimental and we didn't had time to finalize documentation yet. With the Windows version it can be used with SEB Server, which will generate the required SEB settings to connect to a Jitsi Meet server. In the SEB 3.0 iOS version it is possible to manually create SEB settings for connecting to a Jitsi Meet server, these settings can also be used with the 3.2 Windows version (also see the remote proctoring example settings on the page https://demo.safeexambrowser.org/exams/, SEB admin pw in these settings is "seb", quit password "quit").

There is no Android version because someone would need to pay for it to be developed (actually to pay the salary of an additional developer). The current members of the SEB Alliance don't require an Android version. If we get enough funding, we will provide an Android (or ChromeOS or Linux) version.
